specification/storage/resource-manager/Microsoft.Storage/2017-06-01/storage.json

```diff
@@ -700,6 +731,34 @@
       ],
       "description": "The parameters used to check the availabity of the storage account name."
     },
+    "SKUCapability": {
+      "properties": {
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of capability, The capability information in the specified sku, including file encryption, network acls, change notification, etc."
+        },
+        "value": {
```

Comment:
@lmazuel Agreed, too. Boolean is the first thing I tried, but in fact, http response returned a 'string' with value 'true' or 'false' instead of true or false: https://github.com/JasonYang-MSFT/azure-rest-api-specs/blob/storageskus/specification/storage/resource-manager/Microsoft.Storage/2017-06-01/examples/SKUList.json#L69. 
It also failed validation with oav validate-example .\storage.json, message: 'Expected type boolean but found type string'. Though generated .Net code could parse it as boolean, but not sure about other languages.

 This api  exists for a long time, even in 2015-05-01-preview, and it is only in response, so I choose to make it as a string.